addon/components/frost-sort-item.js

```diff
@@ -1,36 +1,58 @@
 import Ember from 'ember'
-import computed from 'ember-computed-decorators'
 import layout from '../templates/components/frost-sort-item'
-import _ from 'lodash/lodash'
+import PropTypesMixin, { PropTypes } from 'ember-prop-types'
 
-const {Component, isEmpty} = Ember
+const {
+  A,
+  computed,
+  Component,
+  isEmpty
+} = Ember
 
-export default Component.extend({
+export default Component.extend(PropTypesMixin, {
   layout: layout,
   classNames: ['frost-sort-item'],
 
-  @computed
```

Comment:
I'm not sure, we'll find out the more we use the `ember-cli-code-coverage` addon fork/release